config/output.go

```diff
@@ -36,7 +36,7 @@ func (t *Config) RunOutputs() (err error) {
 func (t *Config) runOutputs(evchan chan logevent.LogEvent, logger *logrus.Logger) (err error) {
 	outputs, err := t.getOutputs()
 	if err != nil {
-		return errutil.New("get outputs failed", err)
+		return err
```

Comment:
Anyway, let's check that another time... I'll make this PR only about the AMQP output.